app/helpers/compliance_hosts_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ComplianceHostsHelper
+  
+  Colors = {
+    :passed  => '#89A54E',
+    :failed  => '#AA4643',
+    :othered => '#DB843D',
```

Comment:
seems like duplication of ArfReportDashboardHelper::Colors, during fixing, could you please rename Colors to COLORS?